tests/setups/setupTablesFunctionCjs.ts

```diff
@@ -0,0 +1,5 @@
+import { join } from "path";
+import { setup } from "../../src";
+
+// Setup with the config dir
+setup(join(__dirname, "../configs/javascript"));
```

Comment:
You need to make a folder for `configs/cjs` and make a config which is CJS so that it tests CJS works

Reply:
Ok done